ros2controlcli/ros2controlcli/verb/list_hardware_components.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 ROS2-Control Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from controller_manager import list_hardware_components
+from controller_manager.spawner import bcolors
```

Comment:
It may be a better idea to move those colours to a separate file and avoid importing things from spawner by accident. It could be a new issue

Reply:
It looks like `bcolors` is only used in `spawner.py` and `list_hardware_interfaces.py`. I can separate that in a new PR.